Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -72,6 +73,83 @@ Lambda Annotations uses C# source generators to generate that boiler plate code
 In addition the source generator also synchronizes the CloudFormation template to declare all of the .NET methods with the `LambdaFunction` attribute as 
 Lambda functions in the CloudFormation template.
 
+## Getting started
+
+To get started with Lambda annotations a Lambda blueprint is available. For Visual Studio users the blueprint can be 
+accessed using the [AWS Toolkit for Visual Studio](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.AWSToolkitforVisualStudio2022). 
+For non-Visual Studio users the [Amazon.Lambda.Templates](https://www.nuget.org/packages/Amazon.Lambda.Templates) 
+NuGet package is available for creating .NET Lambda projects from the .NET CLI.
+
+
+### Visual Studio 2022
+
+To get started with Visual Studio install [AWS Toolkit for Visual Studio](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.AWSToolkitforVisualStudio2022) 
+extension. Once installed a project can be created with the Lambda Annotations preconfigured using
+the following steps.
+
+* Select **Create a new project**
+* In the template search box enter **AWS Serverless**
+* Select the **AWS Serverless Application (.NET Core C#)** from the search result and click **Next**
+* Name the project and click **Create**
+* The **Select Blueprint** wizard will be displayed for choosing the initial content of the project.
+* Select the **Annotations Framework** blueprint and click **Finish**
+
+### .NET CLI
+
+.NET Lambda projects can be be created using the .NET CLI's `dotnet new` command. To create a project using the
+Lambda Annotations library run the following steps from a terminal.
+
+* Run the following command to install the AWS Lambda templates into the .NET CLI
+  * `dotnet new install Amazon.Lambda.Templates`
+* Run the following command to create a project using Lambda Annotations
+  * `dotnet new serverless.Annotations --output FirstAnnotationsProject` 
+
+This will create a project in a sub directory of the current director called `FirstAnnotationsProject`. The directory 
+will contain both a Lambda project using Annotations as well as a unit test project.
+
+### The sample project
+
+The starting project contains the following files:
+
+* **Functions.cs** - Defines a collection of REST API Lambda functions using Lambda Annotation.
+* **Startup.cs** - Where services can be registered for dependency injection into the Lambda functions.
+* **serverless.template** - CloudFormation template used to deploy the Lambda functions. The Lambda Annotations library 
+will automatically sync the functions defined in the project in the CloudFormation template.
+* **aws-lambda-tools-defaults.json** - Config file for default settings used for deployment.
+
+To reset the project to an empty project delete the code in the `Functions` class and recompile the project. The Lambda
```

Comment:
Do you need "to an empty project"? Lots of "project" in this sentence. "To reset the sample project delete the code ..."

Reply:
I think it is important to say what we are resetting to. I took out the first "project" so it now ways "To reset to an empty project..."